src/main/java/org/mskcc/cmo/NewSamplePublisher.java

```diff
@@ -0,0 +1,98 @@
+package org.mskcc.cmo;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonIOException;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.reflect.TypeToken;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import org.mskcc.cmo.messaging.Gateway;
+import org.mskcc.cmo.shared.SampleMetadata;
+import org.mskcc.cmo.shared.neo4j.Patient;
+import org.mskcc.cmo.shared.neo4j.PatientMetadata;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ *
+ * @author DivyaMadala
+ */
+@SpringBootApplication(scanBasePackages = "org.mskcc.cmo.messaging")
+public class NewSamplePublisher implements CommandLineRunner {
+
+    @Autowired
+    private Gateway messagingGateway;
+    private static int numOfSamples;
+
+    private static String fileName;
```

Comment:
- Read filename from `args` and publish data directly to topic `igo.new-sample-intake`
- Assume the file contains JSON, will not require any validation or manipulation
- Assume only one sample per file
- Application exits after successful publishing to topic